rtos/TARGET_CORTEX/mbed_boot.c

```diff
@@ -430,6 +430,18 @@ void __rt_entry (void) {
  */
 #if defined(RTX_NO_MULTITHREAD_CLIB)
 
+// These 2 macros are taken from RTX Config (internal header)
```

Comment:
That should probably go into our config file?

Reply:
I can move it there with default values set. Based on our config, there would be just one
`#define OS_THREAD_LIBSPACE_NUM      1`, will check

Reply:
Latest rebase fixed it, moved to conf with default value as it was (4)